token_review.go

```diff
@@ -73,7 +73,7 @@ func (t *tokenReviewAPI) Review(jwt string) (*tokenReviewResult, error) {
 	}
 
 	// Build the request to the token review API
-	url := fmt.Sprintf("%s/apis/authentication.k8s.io/v1/tokenreviews", t.config.Host)
+	url := fmt.Sprintf("%s/apis/authentication.k8s.io/v1/tokenreviews", strings.TrimSuffix(t.config.Host, "/"))
```

Comment:
Can you just use https://golang.org/pkg/path/#Join here?

Reply:
We could, I briefly looked for some kind of `join` but I was looking in the `http` package. Interesting enough, trying it out in the playground shows `path.Join` removing one of the `/` from the url scheme

- https://play.golang.org/p/DoHiFCf4N9R

That url seems to work in a browser and curl, but it still seems weird, unless I constructed that example incorrectly. 

Is there any other advantage to changing from the existing `fmt.Sprintf` to `path.Join`?